src/controllers/billsController.ts

```
@@ -84,11 +84,11 @@ export async function postBillsPay(
     });
   } catch (e) {
     if (e instanceof z.ZodError) {
-      next(new AppError(e.errors.map((err) => err.message).join("; "), 400));
-      return;
+      throw new AppError("Validation error", 400, "VALIDATION_ERROR", e.flatten());
```

ESLint

[error] Replace `"Validation·error",·400,·"VALIDATION_ERROR",·e.flatten()` with `⏎········"Validation·error",⏎········400,⏎········"VALIDATION_ERROR",⏎········e.flatten(),⏎······`

(prettier/prettier)